Modules/ThirdParty/Eigen3/src/itkeigen/CMakeLists.txt

```diff
@@ -1,11 +1,40 @@
-# cmake_minimum_require must be the first command of the file
 # Same version of ITK cmake_minimum_required
-cmake_minimum_required(VERSION 3.16.3)
+cmake_minimum_required(VERSION 3.10.2)
 
 project(Eigen3)
 
 # ITK doesn't compile anything here, just generates targets for the INTERFACE library.
 if(FALSE)
+
+# NOTE Remove setting the policy once the minimum required CMake version is
+# increased to at least 3.15. Retain enabling the export to package registry.
+if (POLICY CMP0090)
+  # The export command does not populate package registry by default
+  cmake_policy (SET CMP0090 NEW)
+
+  # Unless otherwise specified, always export to package registry to ensure
+  # backwards compatibility.
+  if (NOT DEFINED CMAKE_EXPORT_PACKAGE_REGISTRY)
+    set (CMAKE_EXPORT_PACKAGE_REGISTRY ON)
+  endif (NOT DEFINED CMAKE_EXPORT_PACKAGE_REGISTRY)
```

Comment:
Is that consistent with the behavior before this update ?

Reply:
That's dead code for us. There is a `if(FALSE)` a few lines before.
This block was added by upstream Eigen, we ignore it.